engine/core/src/luos_engine.c

```diff
@@ -1032,6 +1054,21 @@ uint16_t Luos_NbrAvailableMsg(void)
 {
     return MsgAlloc_LuosTasksNbr();
 }
+/******************************************************************************
+ * @brief check if message of a specific topic is available
+ * @param service pointer
+ * @param topic_id
+ * @return true or false
+ ******************************************************************************/
+uint8_t Luos_FindTopicMsg(service_t *service, uint16_t topic_id)
+{
+    // search in msg alloc if we have a msg_task of a topic for this service
+    if (MsgAlloc_SearchLuosTaskTopic(service->ll_service, 0, topic_id) == SUCCEED)
```

Comment:
Passing 0 to this function will write something at address 0. You should change that.

Reply:
@BiliouriV this is not a comment but a modification request. I think @JeromeGalan missclick on the PR state.
You should fix it to make this PR mergeable.